gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -456,17 +488,22 @@ def main(args):
         action="store_true",
     )
 
-    preprocess_data_args = parser.add_argument_group(
-        "Preprocess data args", "Arguments used for preprocessing the data."
+    prepare_context_args = parser.add_argument_group(
+        "Prepare context Table args", "Arguments used for preparing the context Table."
     )
-    preprocess_data_args.add_argument(
+    prepare_context_args.add_argument(
         "--prepare-context-ht",
         help=(
-            "Whether to split multiallelic sites and add 'methylation', 'coverage', and"
-            " 'gerp' annotation to context Table with VEP annotation."
+            "Prepare the context Table by splitting multiallelic sites and adding "
```

Comment:
aren't multiallelic sites already split?

Reply:
yeah, I was confused about that too, maybe the original one used was not. I can either remove that or add a check to see if it's split and only split if not. Thoughts?

Reply:
i like adding a check and only splitting if necessary